cfme/fixtures/smtp.py

```diff
@@ -38,7 +38,7 @@ def smtp_test(request, appliance):
     logger.info("Mind that it needs ports %s and %s open", mail_query_port, mail_server_port)
     appliance.server.settings.update_smtp_server({
         'host': my_ip,
-        'port': mail_server_port,
+        'port': str(mail_server_port),
```

Comment:
This was causing automation failure - fill was thinking that values were changed and was filling field again => but Save button was grayed and we couldn't pass assertions in flashmessages